wordpress-mobile/GutenbergKit
# Fix Editor Focus issues

I originally thought that the editor text selection issues were released to the WebView not being the first responder on iOS, but that's not the case – it seems this issue has been addressed upstream, so pulling in the latest version of the dependencies corrects it.

To test:

1. Run this project from `trunk` on-device and paste a large amount of text. Note that it's difficult to select a small portion of text, and attempting to use "Select All" is difficult to invoke. 
2. Run this PR on-device and try the test again. It should now work how you expect.
